packages/library/src/common/runtime/serialize.library.ts

```
@@ -0,0 +1,375 @@
+			return new Date(value);
+		case 'RegExp':
+			return new RegExp(value.source, value.flags);
+		case 'Symbol':
+			return toSymbol(value);
+		case 'Map':
+			return new Map(value);
+		case 'Set':
+			return new Set(value);
+
+		default:
+			const cls = Registry.get($type);											// lookup registered Class
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)